Neos.Flow/Configuration/Objects.yaml

```diff
@@ -149,6 +149,15 @@ Neos\Flow\Log\SecurityLoggerInterface:
     4:
       setting: Neos.Flow.log.securityLogger.backendOptions
 
+Neos\Flow\Log\ThrowableLoggerInterface:
+  scope: singleton
+  className: Neos\Flow\Log\ThrowableStorage\FileStorage
```

Comment:
This feels strange… How can a storage be a logger? A logger should be _using_ a storage, not be one. No?